cubes/formatters.py

```diff
@@ -2,18 +2,22 @@
 
 from __future__ import print_function
 
-import json
-import csv
 import codecs
-import decimal
+import csv
 import datetime
-
+import decimal
+import json
+import os
+import tempfile
 from collections import namedtuple
 
+from openpyxl import Workbook
```

Comment:
This should be changed to "protected" import as the `jinja2` below so we don't push hard dependency on the `openpyxl` library.